app/connectors/emcsTfe/GetMovementListConnector.scala

```diff
@@ -32,8 +33,8 @@ class GetMovementListConnector @Inject()(val http: HttpClient,
   override implicit val reads: Reads[GetMovementListResponse] = GetMovementListResponse.reads
 
   lazy val baseUrl: String = config.emcsTfeBaseUrl
-  def getMovementList(exciseRegistrationNumber: String)
+  def getMovementList(exciseRegistrationNumber: String, search: Option[MovementListSearchOptions] = None)
                      (implicit headerCarrier: HeaderCarrier, executionContext: ExecutionContext): Future[Either[ErrorResponse, GetMovementListResponse]] =
-    get(s"$baseUrl/movements/$exciseRegistrationNumber")
+    get(s"$baseUrl/movements/$exciseRegistrationNumber", search.fold[Seq[(String, String)]](Seq.empty)(_.queryParams))
```

Comment:
I feel like folds on an option are overkill and less easy to understand than other approaches
```suggestion
    get(s"$baseUrl/movements/$exciseRegistrationNumber", search.map(_.queryParams)).getOrElse(Seq.empty)
```